.changeset/sixty-pianos-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/button': major
+---
+
+Component now composed of React.forwardRef()
```

Comment:
nit: Wrapping component with React.forwardRef to provide direct access to the underlying element.

Reply:
👍